features/gezag-minderjarige/geen-gezag.feature

```diff
@@ -0,0 +1,21 @@
+#language: nl
```

Comment:
We zouden ervoor kunnen kiezen om een aantal gegevens op te nemen in de achtergrond. Zo voorkomen we dat velden zoals gemeente van inschrijving leeg zijn.

```suggestion
#language: nl

Functionaliteit: Over meerderjarige of overleden personen wordt geen gezag geleverd

Achtergrond:
 Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
  * is meerderjarig
  * is in Nederland geboren
  * is ingeschreven in de BRP

  Regel: Als de persoon meerderjarig is, dan is er geen sprake van gezag

    Scenario: De persoon is meerderjarig
      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
      Dan heeft deze persoon geen gezaghouder


  Regel: Als de persoon overleden is, dan is er geen sprake van gezag

    Scenario: De persoon is overleden
      Gegeven persoon 'Bert'
      * is overleden
      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
      Dan heeft deze persoon geen gezaghouder
```